docs/source/delta-row-tracking.md

```diff
@@ -0,0 +1,113 @@
+
+  .. code-language-tabs::
+    ```sql
+    SET spark.databricks.delta.properties.defaults.enableRowTracking = true;
+    ```
+
+    ```python
+    spark.conf.set("spark.databricks.delta.properties.defaults.enableRowTracking", True)
+    ```
+
+    ```scala
+    spark.conf.set("spark.databricks.delta.properties.defaults.enableRowTracking", true)
+    ```
+
+.. important:: Because cloning a <Delta> table creates a separate history, the row ids and row commit versions on cloned tables do not match that of the original table.
+
+.. important:: Enabling row tracking on existing table will automatically assign row ids and row commit versions to all existing rows in the table. This process may cause multiple new versions of the table to be created and may take a long time.
+
+.. warning:: Tables created with row tracking enabled have the row tracking <Delta> table feature enabled at creation and use <Delta> writer version 7. Table protocol versions cannot be downgraded, and tables with row tracking enabled are not writeable by <Delta> clients that do not support all enabled <Delta> writer protocol table features. See [_](/versioning.md).
+
+### Row tracking storage
+
+Enabling row tracking may increase the size of the table. <Delta> stores row tracking metadata fields in hidden metadata columns in the data files. Some operations, such as insert-only operations do not use these hidden columns and instead track the row ids and row commit versions using metadata in the <Delta> log. Data reorganization operations such as `OPTIMIZE` and `REORG` cause the row ids and row commit versions to be tracked using the hidden metadata column, even when they were stored using metadata.
+
+## Read row tracking metadata fields
+
+The row ids and row commit versions metadata fields are not automatically included when reading the table. Instead, these metadata fields must be manually selected from the hidden `_metadata` column which is available for all tables in <AS>.
+
+.. code-language-tabs::
+  ```sql
+  SELECT _metadata.row_id, _metadata.row_commit_version, * FROM table_name;
+  ```
+
+  ```python
+  spark.read.table("table_name") \
+    .select("_metadata.row_id", "_metadata.row_commit_version", "*")
+  ```
+
+  ```scala
+  spark.read.table("table_name")
+    .select("_metadata.row_id", "_metadata.row_commit_version", "*")
+  ```
+
+## What is the schema of the row tracking metadata fields?
+
+Row tracking adds the following metadata fields that can be accessed when reading a table:
+
+| Column name                    | Type | Values                                                                       |
+|--------------------------------|------|------------------------------------------------------------------------------|
+| `_metadata.row_id`             | Long | The unique identifier of the row.                                            |
+| `_metadata.row_commit_version` | Long | The table version at which the row was last inserted or updated. |
```

Comment:
I would merge these two sections. Start with describing the metadata fields and then considerations about accessing the hidden `_metadata` column